<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>499</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Play">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton  {
    background-color: rgb(0, 255, 127);
    color:white;
    border-style: outset;
    border-width: 5px;
    border-radius: 10px;
    border-color: rgb(0, 255, 127)
}

QPushButton:pressed {
     background-color:rgb(49, 49, 49);
     border-color:rgb(49, 49, 49);
}
</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   background-color: rgb(220, 211, 255);
   color: rgb(191, 79, 255);
   border-style: outset;
   border-width: 5px;
   border-radius: 10px;
   border-color: rgb(205, 180, 255)
}
</string>
    </property>
    <property name="text">
     <string>Выберите индекс микрофна(первое число строки)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   background-color: rgb(0, 255, 127);
   color: rgb(0, 121, 56);
   border-style: outset;
   border-width: 5px;
   border-radius: 10px;
   border-color: rgb(0, 255, 127)
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   background-color: rgb(220, 211, 255);
   color: rgb(191, 79, 255);
   border-style: outset;
   border-width: 5px;
   border-radius: 10px;
   border-color: rgb(205, 180, 255)
}
</string>
    </property>
    <property name="text">
     <string>Задайте время записи в секундах</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   background-color: rgb(220, 211, 255);
   color: rgb(191, 79, 255);
   border-style: outset;
   border-width: 5px;
   border-radius: 10px;
   border-color: rgb(205, 180, 255)
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   background-color: rgb(180, 180, 180);
   border-style: outset;
   border-width: 5px;
   border-radius: 10px;
   border-color: rgb(84, 84, 84)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>600</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   background-color: rgb(0, 255, 127);
   color: rgb(0, 121, 56);
   border-style: outset;
   border-width: 5px;
   border-radius: 10px;
   border-color: rgb(0, 255, 127)
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>Play</zorder>
   <zorder>listWidget</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBox_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>spinBox_3</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
